pkg/common/peertracker/tracker_bsd.go

```diff
@@ -9,6 +9,13 @@ import (
 	"sync"
 	"syscall"
 	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spiffe/spire/pkg/common/telemetry"
+)
+
+const (
+	_bsdType = "bsd"
```

Comment:
nit: this naming does not follow the conventions for unexported package level consts in the codebase
```suggestion
	bsdType = "bsd"
```